boards/xtensa/esp32s3/esp32s3-eye/include/board.h

```diff
@@ -43,4 +43,10 @@
 
 #define BOARD_NLEDS             1
 
+/* GPIO pins used by the GPIO Subsystem */
+
+#define BOARD_NGPIOOUT    2 /* Amount of GPIO Output pins */
```

Comment:
Do you need these two pins to be exposed as GPIO for your system for a specific reason? Usually, we only enable one pin for demonstration purposes to avoid it being "fixed" as a GPIO.

Reply:
I would normally prefer to control GPIO48 with PWM/LEDC, but it is not working so far. So i enabled as GPIO so at least we have some control for it.

If you prefer I can drop the GPIO48 and enable only GPIO3.

Or we can do it when we finally get the PWM to work.